generator/src/main/java/com/scottlogic/deg/generator/generation/SedolStringGenerator.java

```diff
@@ -11,6 +11,7 @@
 // There's a lot of copy-paste in this class. We should address that before we make any significant changes to it
 public class SedolStringGenerator implements StringGenerator {
 
+    public static final int SEDOL_LENGTH = 7;
```

Comment:
This is true of SEDOLs in the real world, but not the ones generated by this (faulty) class, which worries me. Is there any code that relies on this thing generating 7-character strings?

Reply:
This could be removed as we don't access/produce sedol codes in the Generator yet

Reply:
Removed